pages/components/Logo/styles.module.scss

```diff
@@ -0,0 +1,217 @@
+@use "sass:map";
+@use "app/var.module";
+
+// $upper-gap-height: 12px;
+// $lower-gap-height: 24px;
+$upper-gap-height: 1.35%; // as a % of the width of the inner hex
+$lower-gap-height: 2.70%; // as a % of the width of the inner hex
+// $outer-hex-diff-horizontal: 18px;
+// $outer-hex-diff-vertical: 19px;
+$outer-hex-diff-horizontal: 1.94%; // as a % of the width of the outer container
+$outer-hex-diff-vertical: 1.8%; // as a % of the height of the outer container
+
+// Height = 1.15 x Width (use Math.round)
+// Original width: 926px;
+// Original height: 1065px;
+
+.logoContainer {
+  position: relative;
+  margin: 0 auto;
```

Comment:
This should not be determined by the `Logo` component. The consumer of the component should determine positioning.